src/components/sidecar/style.module.css

```diff
@@ -1,11 +1,5 @@
 .sidecar {
-  margin-left: 24px;
-  max-width: 220px;
-  min-width: 220px;
-  position: sticky;
   text-align: right;
-  top: 0;
-  width: 220px;
```

Comment:
Positioning and width styles no longer needed, `sidebar-sidecar-layout` handles these, and we just need to drop in content.